### ISSUE TITLE
Pipelines and Colored Text for Pre-Processing

### ISSUE THREAD

**Issue by knpwrs** [maaslalani/slides#148]
Is it possible to support pipelines for pre-processing? I was hoping to use the following command to make a slide:

```
echo prrm | figlet -f broadway | boxes -a c -d unicornsay | lolcat
```

But it doesn't appear to work. I can put it in a file called `prrm.sh` with `#!/usr/bin/env zsh` and then use `./prrm.sh`.

Also, while running `./prrm.sh` results in the following:

![image](https://user-images.githubusercontent.com/174864/160922467-0f4a7ccd-1825-4fc0-8953-ff9d2064a319.png)

When displayed through `slides` I get no color:

![image](https://user-images.githubusercontent.com/174864/160922533-7b42ecf8-7c35-413f-95c5-7160730a5a4e.png)

**Comment by maaslalani** [comment]
Hey! Thanks so much for taking the time to create this issue, I will try to take a look. The pipe stuff should be solvable by doing exactly what you did just in the background / invisible to the user. I believe the colouring stuff should be fixable as well :)

**Comment by maaslalani** [comment]
@knpwrs Btw, this may be because `lolcat` detects that there is no tty and prints out the art without color. Similar as to what would happen if you were to `lolcat | pbcopy`.

Can you try:

```
lolcat --force
```

to force color output.

**Comment by knpwrs** [comment]
`lolcat --force` does the trick! Thank you!

I'm going to leave this issue open assuming there isn't a fix yet for pipes. If there is one already feel free to close this!

**Comment by maaslalani** [comment]
Yup, that's correct, I don't think anything has changed with the pipes, but the solution will likely look similar to converting the command (with pipes) to a shell script in the background.
